src/app-layout/__tests__/desktop.test.tsx

```diff
@@ -253,17 +253,17 @@ describe('Classic only features', () => {
     const { wrapper } = renderComponent(
       <AppLayout toolsHide={true} drawers={[testDrawer]} ariaLabels={{ drawers: 'Drawers' }} />
     );
-    act(() => screen.getByLabelText('Drawers').click());
+    fireEvent.click(screen.getByLabelText('Drawers'));
```

Comment:
What is the benefit of using `fireEvent.click` over `click` in this case?

BTW, it might worth to consider `user-event` lib as an alternative for `fireEvent`, but it's definitely out of this PR scope
https://testing-library.com/docs/guide-events/#alternatives

Reply:
`.click()` is `HTMLElement.prototype.click` without any extra logic. Both our test-utils' `wrapper.click()` and `fireEvent.click()` provide built-in `act` integration so extra wrapper is not needed here.

Using `fireEvent.click` is not ideal, because we recommend everyone using our test utils and here we undermine our own recommendation. Using `userEvent.click` moves us even further from our own test-utils, so definitely not recommended

Reply:
If you have more questions, I think we can chat more in our team channel for better knowledge sharing